static/app/utils/useProjects.tsx

```diff
@@ -150,7 +150,7 @@ async function fetchProjects(
 function useProjects({limit, slugs, orgId: propOrgId}: Options = {}) {
   const api = useApi();
 
-  const {organization} = useLegacyStore(OrganizationStore);
```

Comment:
Swapped this out so I dont need to await for store updates in the tests to fire, not sure if there are any changes here?